app/routes/joblistings/components/JoblistingRightNav.tsx

```diff
@@ -61,64 +47,49 @@ const FilterLink = ({
   );
 };
 
-type Filter = {
-  grades: Array<string>;
-  jobTypes: JobType[];
-  workplaces: Array<string>;
-};
```

Comment:
Is there a reason this is not kept and used to type `updateFilters()`?

Reply:
Just didn't want to refactor too much, it was currently not needed where it was being used because of inferred types